CMakeLists.txt

```diff
@@ -110,32 +113,32 @@ endif()
 
 # Checking sanitizer options; in both cases we include 'undefined'
 set(NETWORKIT_CXX_SANITIZERS "")
-if ("${NETWORKIT_WITH_SANITIZERS}" STREQUAL "address")
+
+if("${NETWORKIT_WITH_SANITIZERS}" STREQUAL "address")
 	set(NETWORKIT_CXX_SANITIZERS "address,undefined")
-elseif ("${NETWORKIT_WITH_SANITIZERS}" STREQUAL "leak")
+elseif("${NETWORKIT_WITH_SANITIZERS}" STREQUAL "leak")
 	set(NETWORKIT_CXX_SANITIZERS "address,leak,undefined")
 elseif(NOT "${NETWORKIT_WITH_SANITIZERS}" STREQUAL "")
 	message(FATAL_ERROR "Unsupported option ${NETWORKIT_WITH_SANITIZERS}")
 endif()
 
-if (NOT "${NETWORKIT_CXX_SANITIZERS}" STREQUAL "")
-    set(NETWORKIT_CXX_FLAGS "-fsanitize=${NETWORKIT_CXX_SANITIZERS} ${NETWORKIT_CXX_FLAGS}")
-    set(NETWORKIT_LINK_FLAGS "-fsanitize=${NETWORKIT_CXX_SANITIZERS} ${NETWORKIT_LINK_FLAGS}")
+if(NOT "${NETWORKIT_CXX_SANITIZERS}" STREQUAL "")
+	set(NETWORKIT_CXX_FLAGS "-fsanitize=${NETWORKIT_CXX_SANITIZERS} ${NETWORKIT_CXX_FLAGS}")
+	set(NETWORKIT_LINK_FLAGS "-fsanitize=${NETWORKIT_CXX_SANITIZERS} ${NETWORKIT_LINK_FLAGS}")
 endif()
 
 # Check if coverage support is enabled
-if (NETWORKIT_COVERAGE)
-  set(NETWORKIT_CXX_FLAGS "${NETWORKIT_CXX_FLAGS} --coverage")
-  set(NETWORKIT_LINK_FLAGS "${NETWORKIT_LINK_FLAGS} --coverage")
+if(NETWORKIT_COVERAGE)
```

Comment:
Would be better to make a special format PR for changes as such :smiley_cat:

Reply:
You are right, I will revert the style changes for `CMakeLists.txt` (since it was not formatted before).